frontend/src/lib-customizations/common.tsx

```diff
@@ -50,4 +52,19 @@ export const absenceColours = {
   PRESENCE: colors.greyscale.white
 }
 
+/* eslint-disable @typescript-eslint/no-explicit-any */
+export const translationsMergeCustomizer = (
```

Comment:
This is the most important change: allows overriding JSX elements in city specific implementations.

Reply:
You can avoid the `eslint-disable` like this:

```
 export const translationsMergeCustomizer = (
  origValue: Record<string, unknown>,
  customizedValue: unknown
): unknown => {
```

Reply:
Maybe even better to have it like this?

```
export const translationsMergeCustomizer = (
  origValue: Record<string, unknown>,
  customizedValue: Record<string, unknown>
): Record<string, unknown> | undefined => {
```

Reply:
Yeah, sure 👍